### PR TITLE
Adds trueblocks-core to resolve ENS names

### PR DESCRIPTION
**Pull request by tjayrush**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -21,7 +27,21 @@ func (a *App) startup(ctx context.Context) {
 	a.ctx = ctx
 }
 
-// Greet returns a greeting for the given name
-func (a *App) Greet(name string) string {
-	return fmt.Sprintf("Hello %s, It's show time!", name)
+// Export returns the number of transactions found at this address
+func (a *App) Export(addressOrEns string) string {
+	conn := rpc.NewConnection("mainnet", false, map[string]bool{})
+	addr, ok := conn.GetEnsAddress(addressOrEns)
```
Comment (naterush) [inline app.go]: Does this work if I pass in an address?

File: app.go [hunk 2]
```diff
@@ -21,7 +27,21 @@ func (a *App) startup(ctx context.Context) {
...
+	if !ok {
+		return fmt.Sprintln("GetEnsAddress returned not okay")
+	}
+	address := base.HexToAddress(addr)
+	cmd := "chifra list --count " + address.Hex() + " --fmt csv --no_header --output file.fil"
```
Comment (naterush) [inline app.go]: Hack: I could pass in something here that runs a separate command, when encoded to hex.

Should do some validation here. Wanna lead a TODO in the code at least?

File: app.go [hunk 3]
```diff
@@ -21,7 +27,21 @@ func (a *App) startup(ctx context.Context) {
...
+	_ = utils.System(cmd)
+	value := file.AsciiFileToString("file.fil")
```
Comment (naterush) [inline app.go]: What is .fil?

File: frontend/wailsjs/runtime/runtime.d.ts [hunk 4]
```diff
@@ -21,8 +21,8 @@ export interface Size {
 export interface Screen {
```
Comment (naterush) [inline frontend/wailsjs/runtime/runtime.d.ts:21]: Wondering if we want to commit this

**Review by naterush — COMMENTED** [review]
Minor code notes. Let me test manually, then LGTM.

**Comment by naterush** [comment]
I think we want to add `frontend/wailsjs` to .gitignore. I get all sorts of nonsense when I build and then try and switch branches.

File: app.go (hunk 1 above)
Comment (tjayrush) [reply app.go]: Yes. It works for both ENS names and addresses.

File: app.go (hunk 3 above)
Comment (tjayrush) [reply app.go]: Changed in ext PR

File: frontend/wailsjs/runtime/runtime.d.ts (hunk 4 above)
Comment (tjayrush) [reply frontend/wailsjs/runtime/runtime.d.ts:21]: We do not

File: app.go (hunk 1 above)
Comment (naterush) [reply app.go]: In my basic test, looks like this didn't work with addresses. 

Additionally, the ENS name resolution seemed a bit wonky. 

Is there some way we can add tests in pure golang for this function?

Comment (tjayrush) [reply app.go]: Tests added.

File: app.go (hunk 2 above)
Comment (tjayrush) [reply app.go]: Should now dissallow anything but valid addresses or valid ens names

**Pull request merged**